src/test/java/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package strategy;
+
+import db.Storage;
+import java.util.Map;
+import model.Fruit;
+import model.TransactionDto;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private static OperationHandler operator;
+    private static Map<Fruit, Integer> expected;
+    private static Fruit fruit;
+    private static Map<Fruit, Integer> actual;
+    private static TransactionDto transactionDto;
```

Comment:
```suggestion
    private static OperationHandler operator;
```
this is hard to understand - let's please make all other variables local